sample/js/main.js

```diff
@@ -1,17 +1,6 @@
-CKEDITOR.config.extraPlugins = 'structuredheadings';
-CKEDITOR.config.contentsCss = './css/numbering.css';
-CKEDITOR.config.allowedContent = true;
-
-var initSample = ( function() {
-
-	var isStructuredHeadingsAvailable = () => {
-		return !!CKEDITOR.plugins.get('structuredheadings');
-	};
-
-	return function() {
-		var editorElement = CKEDITOR.document.getById( 'editor' );
-		CKEDITOR.replace( 'editor');
-	};
-
-})();
+/* exported initSample */
+CKEDITOR.config.extraPlugins = "structuredheadings";
 
+var initSample = function () {
+  CKEDITOR.replace("editor");

```

Comment:
I'm +0 on either, because this is really just a demo page for the plugin.  just weren't sure if you were aware